Pages/data.js

```diff
@@ -0,0 +1,22 @@
+class Data{
+    constructor(){
+        this.d= new Date(),
+        this.months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"],
+        this.homepageLink = "https://auct-app2-frontend.herokuapp.com/",
+        this.EC = protractor.ExpectedConditions,
+        this.headerText= ".header-text",
+        this.emailLocator = "#email",
+        this.userRafaNadal = "rafa@mail.com",
+        this.passwordUserRafaNadal ="password",
+        this.categoryItemLocator=".category-item",
+        this.landingProductLocator=".landing-product",
+        this.bidForSmokeTest=1003,
+        this.biderNameImgLocator=".bider-name-img",
```

Comment:
I would not keep locators here. This class should be used only for data.
Locators should be present in getters inside page objects.

Reply:
I removed locator from data.js, I added getters and method for those locators